src/Avalonia.Controls/SplitButton/ToggleSplitButton.cs

```diff
@@ -0,0 +1,147 @@
+﻿using System;
+using Avalonia.Controls.Metadata;
+using Avalonia.Controls.Primitives;
+using Avalonia.Interactivity;
+using Avalonia.Styling;
+
+namespace Avalonia.Controls
+{
+    /// <summary>
+    /// A button with primary and secondary parts that can each be pressed separately.
+    /// The primary part behaves like a <see cref="ToggleButton"/> with two states and
+    /// the secondary part opens a flyout.
+    /// </summary>
+    [PseudoClasses(
```

Comment:
Will ToggleSplitButton properly inherit this PseudoClass metadata from SplitButton? If so this can be removed.

Reply:
I will. In fact, this attribute is not used by Avalonia, it's only for Rider xaml editor as for now.

Reply:
Ok, I'll leave it in for Rider (will remove from ToggleSplitButton though). 

Separately, is there a TemplatePart attribute? Such as used in WPF/UWP to indicate the parts of the default style that must exist?

Reply:
No

Reply:
Should there be in the future? Should I add an issue for that?

Reply:
Filed a new issue and will add the template parts in the future depending on how that discussion goes.

https://github.com/AvaloniaUI/Avalonia/issues/7432

Reply:
Should be resolved:
 1. PseudoClasses attribute removed from ToggleSplitButton
 2. New issue #7432 added for TemplatePart attribute